packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -1342,6 +1344,37 @@ abstract class FlutterCommand extends Command<void> {
     );
   }
 
+  // This adds the Dart defines used to access various Flutter version information at runtime.
+  void _addFlutterVersionToDartDefines(List<String> dartDefines) {
+    const List<String> flutterVersionDartDefines = <String>[
+      'FLUTTER_VERSION',
+      'FLUTTER_CHANNEL',
+      'FLUTTER_GIT_URL',
+      'FLUTTER_FRAMEWORK_REVISION',
+      'FLUTTER_ENGINE_REVISION',
+      'FLUTTER_DART_VERSION',
+    ];
```

Comment:
there's duplication of these strings here -- we should use a single constant that is in both `flutterVersion` and `flutterVersionDartDefines` so that if someone changes just one the other one is not left dangling.

Reply:
Done